pkg/aws/apis/validation/validation.go

```diff
@@ -123,10 +123,9 @@ func validateBlockDevices(blockDevices []awsapi.AWSBlockDeviceMappingSpec, fldPa
 			allErrs = append(allErrs, field.Required(idxPath.Child("ebs.volumeSize"), "Please mention a valid EBS volume size"))
 		}
 
-		if disk.Ebs.VolumeType == awsapi.VolumeTypeIO1 && disk.Ebs.Iops <= 0 {
+		if disk.Ebs.Iops < 0 || (disk.Ebs.VolumeType == awsapi.VolumeTypeIO1 && disk.Ebs.Iops == 0) {
```

Comment:
```suggestion
		if disk.Ebs.Iops <= 0 || (disk.Ebs.VolumeType == awsapi.VolumeTypeIO1 && disk.Ebs.Iops == 0) {
```

Reply:
`disk.Ebs.Iops = 0` means that the IOPS field was not set in the configuration. This is invalid only in the case of IO1 volumes and not for GP3.

Reply:
hmm. right, then pls ignore this suggestion